ros-dev-tools/README.md

```diff
@@ -0,0 +1,11 @@
+# ROS Dev Tools
+
+`ros-dev-tools` contains packages that are of general use to ROS developers but which are not dependencies of any particular packages within a ROS distribution.
+
+### Packages included in this variant
+
+* ros-build-essential
+* [rosdep](https://github.com/ros-infrastructure/rosdep)
+* [colcon-common-extensions](https://github.com/colcon/colcon-common-extensions)
+* [bloom](https://github.com/ros-infrastructure/bloom)
+* [vcstool](https://github.com/dirk-thomas/vcstool)
```

Comment:
Nit: alphabetize this list, just so that it matches the alphabetization in the control/spec files?  Really nitpicking here, feel free to ignore.

Reply:
Done in [2467a2f](https://github.com/ros-infrastructure/infra-variants/pull/1/commits/2467a2f12b073d69e2e7dab5a053c732e957ef96)